### PR TITLE
Add magic function to do the magic things.

### PR DESCRIPTION
**Pull request by elijahtsai**
The magic function will auto-download the latest app from the official website, patching and push to Applications folder.

### CODE REVIEW

**Comment by duraki** [comment]
LGTM but I wouldn't push to master yet until we test it thoroughly. Please create a new branch and push to `feature-magic`.

**Comment by elijahtsai** [comment]
Please send the script to Big Sur and check the compatibility.

File: sketchcrapp.sh [hunk 1]
```diff
@@ -527,8 +544,9 @@ magicFunction() {
   curl "$latestBundleURLPath" --output "/tmp/latest.zip"
 
   if ! [ "$?" -eq "0" ]; then
-    echo "Fail to download latest application."
-    echo "Are you up to the Internet??"
+    echo "[-] Fail to download latest application."
```
Comment (duraki) [inline sketchcrapp.sh:547]: Failed while downloading latest application version!

File: sketchcrapp.sh [hunk 2]
```diff
@@ -527,8 +544,9 @@ magicFunction() {
...
+    echo "[-] Are you up to the Internet??"
```
Comment (duraki) [inline sketchcrapp.sh:548]: Are you connected to the internet? Check your network connection.

File: sketchcrapp.sh [hunk 3]
```diff
@@ -537,7 +555,7 @@ magicFunction() {
     echo "Exist. Removing."
     rm -rf "/tmp/Sketch.app"
     if ! [ "$?" -eq "0" ]; then
-      echo "Fail to remove exist Sketch.app in /tmp directory."
+      echo "[-] Fail to remove exist Sketch.app in /tmp directory."
```
Comment (duraki) [inline sketchcrapp.sh:558]: Can't remove existing Sketch.app from /tmp directory.

File: sketchcrapp.sh [hunk 4]
```diff
@@ -548,7 +566,7 @@ magicFunction() {
   unzip -q "/tmp/latest.zip" -d "/tmp"
 
   if ! [ "$?" -eq "0" ]; then
-    echo "Fail to unzip zip file of latest application."
+    echo "[-] Fail to unzip zip file of latest application."
```
Comment (duraki) [inline sketchcrapp.sh:569]: Can't unzip downloaded archived file of the latest application version.

File: sketchcrapp.sh [hunk 5]
```diff
@@ -572,7 +590,7 @@ magicFunction() {
 
   if ! [ "$?" -eq "0" ]; then
     echo "Fail"
-    echo "Fail to moving /tmp/Sketch.app to /Applications directory."
+    echo "[-] Fail to moving /tmp/Sketch.app to /Applications directory."
```
Comment (duraki) [inline sketchcrapp.sh:593]: Failed while moving /tmp/Sketch.app to /Applications directory.

**Review by duraki — COMMENTED** [review]
WDYT

**Comment by elijahtsai** [comment]
> WDYT

Nice suggestion, I have implemented.

**Comment by elijahtsai** [comment]
Also, I think we need to let them know this feature in the readme file.

**Comment by elijahtsai** [comment]
and if everything going well we need to push to master branch right?

**Review by duraki — COMMENTED** [review]
Awesome! Thanks Elijah <3

**Pull request merged**